MVC/controller.js

```diff
@@ -10,3 +10,13 @@ exports. getTopics = (req, res, next) => {
 exports. getAPI = (req, res, next) => {
     res.status(200).send(fetchAPI())
 }
+
+exports. getArticle = (req, res, next) => {
+    const { article_id } = req.params
+    fetchArticle(article_id)
+    .then((article) => {
+        res.status(200).send(article)
```

Comment:
Any data sent from a controller needs to be sent back on a meaningful key ie. .send({article})